pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/documentation/CommentSizeRule.java

```diff
@@ -59,29 +47,14 @@ public Object visit(ASTCompilationUnit cUnit, Object data) {
                     + ": Too many lines", comment.getBeginLine(), comment.getEndLine());
             }
 
-            List<Integer> lineNumbers = overLengthLineIndicesIn(comment);
-            if (lineNumbers.isEmpty()) {
-                continue;
-            }
-
-            int offset = comment.getBeginLine();
-            for (int lineNum : lineNumbers) {
-                int lineNumWithOff = lineNum + offset;
-                addViolationWithMessage(
-                    data,
-                    cUnit,
-                    this.getMessage() + ": Line too long",
-                    lineNumWithOff,
-                    lineNum
```

Comment:
this was therefore a zero, which causes an assertion error when assertions are enabled because line numbers are 1-based